app/services/identity_linker.rb

```diff
@@ -43,9 +43,7 @@ def identity
   end
 
   def find_or_create_identity_with_costing
-    identity_record = identity_relation.first
```

Comment:
Is the `identity_relation` method unused after this?

Reply:
ah great catch, it is not. Removed!